sql/core/src/main/scala/org/apache/spark/sql/execution/command/views.scala

```diff
@@ -255,3 +287,43 @@ case class AlterViewAsCommand(
     session.sessionState.catalog.alterTable(updatedViewMeta)
   }
 }
+
+/**
+ * Drops a view which matches the given name.
+ *
+ * If the view name contains database prefix, this command will drop a permanent view matching the
+ * given name, or throw an exception if view not exist. Else, this command will try to drop a local
+ * temporary view first, if view not exist, try global temporary view next, if still not exist, try
```

Comment:
Hi, @cloud-fan .

Then, is there no way to drop a global temporary view when there exists a local temporary view?

e.g. If I created the followings,
1. default.view1 (permanent view in `default` schema)
2. view1 (local temporary view).

And, suddenly, a global view is created by another session.
3. view1 (global temporary view)

In my session, is there any way to drop 3 (global view view1) without dropping 2 (my local view 2)?